draft-ietf-rats-corim.md

```diff
@@ -1300,6 +1300,35 @@ they are not required to use the same internal data structures.
 For example, it is expected that the resources used during the initialisation
 phase can be amortised across multiple appraisals.
 
+## Verifier Abstraction
```

Comment:
Thank you for the text. I am wondering perhaps this paragraph fits more logically at start (i.e. Line 1284) before we start enumerating phases of Verification ?

Reply:
Either location works for me